docs/intro/intro_scaling.rst

```diff
@@ -93,17 +101,15 @@ Let's see how long the construction takes.
 
 .. note::
 
-    Runtimes are highly system dependent. The numbers here are meant to be
-    representative. You may get different values when you run them on your
-    own system.
+    Because runtimes are highly system dependent, running the code on your system
+    will likely result in different values. The results shown here are illustrative.
 
-Use quicksum
-============
+Summation
```

Comment:
I much prefer the imperative form of these section titles. "Summation" doesn't tell me very much, and is in fact not really relevant to the specific advice.